### ISSUE TITLE
Feature Request Better Score Screen/Board

### ISSUE THREAD

**Issue by MancusNemo** [PadWorld-Entertainment/worldofpadman#159]
Hello,

why not upgrade the Score Screen/Board.
At the moment you see only Points/Scores and your Playtime after a round on a Map. Not the Kills and Death and the won Medals.
Why do I get Medals when I don't see it in the Score table/Score Board at the end of every Round?
Used ammo and Hitting ammo and as consequence calculated accuracy. And what is with your Lolly Points or your Logosprays?
And do not forgot to think about the new Mods like Freezetag.

Can you do there anything?

**Comment by kai-li-wop** [comment]
WoP doesn't focus on the number of "kills" and "deaths", so we don't want to emphasize that too much during the matches. In FFA you can see your points, which is basically the number of "kills". Scoring points in SYC are also visible, but not in the team game types. WoP 1.7 will provide a display of individual scores in team game types in the scoreboard (already implemented). A better support of displaying medals in the scoreboard is also planned and will hopefully be delivered with WoP 1.7 (see #194). In general, the scoreboard has limited space and we need to be careful not to overload it.

What you are asking for looks to me like a player stats feature to be integrated into the game's menu. If you want to bring ideas for a presentation of player stats, please write it in a new ticket, as this will have nothing to do with the scoreboard then.

**Comment by MancusNemo** [comment]
Hello,

I opened the ticket because I was asked again and again how are the points calculated. And why do we get medals when they are not displayed at the end of an round..

A precise statistic would be exaggerated. That is why I did not write it that way. It would also not be desired.

That's why I had the idea to open a ticket to revise the end-of-round screen in it, which also answers these questions.

I hope now that I could express myself more clearly and understandably.
